vision/google/cloud/vision/color.py

```diff
@@ -26,20 +26,38 @@ def __init__(self, colors):
         self._colors = colors
 
     @classmethod
-    def from_api_repr(cls, response):
+    def from_api_repr(cls, image_properties):
         """Factory: construct ``ImagePropertiesAnnotation`` from a response.
 
-        :type response: dict
-        :param response: Dictionary response from Vision API with image
-                         properties data.
+        :type image_properties: dict
+        :param image_properties: Dictionary response from Vision API with image
+                                 properties data.
 
-        :rtype: :class:`~google.cloud.vision.color.ImagePropertiesAnnotation`.
-        :returns: Populated instance of ``ImagePropertiesAnnotation``.
+        :rtype: list of
+                :class:`~google.cloud.vision.color.ImagePropertiesAnnotation`.
+        :returns: List of ``ImagePropertiesAnnotation``.
         """
-        raw_colors = response.get('dominantColors', {}).get('colors', ())
-        colors = [ColorInformation.from_api_repr(color)
-                  for color in raw_colors]
-        return cls(colors)
+        colors = image_properties.get('dominantColors', {}).get('colors', ())
+        return cls([ColorInformation.from_api_repr(color)
+                    for color in colors])
+
+    @classmethod
+    def from_pb(cls, image_properties):
+        """Factory: construct ``ImagePropertiesAnnotation`` from a response.
+
+        :type image_properties: :class:`~google.cloud.grpc.vision.v1.\
+                                image_annotator_pb2.ImageProperties`
+        :param image_properties: Protobuf response from Vision API with image
+                                 properties data.
+
+        :rtype: list of
+                :class:`~google.cloud.vision.color.ImagePropertiesAnnotation`
+        :returns: List of ``ImagePropertiesAnnotation``.
+        """
+        colors = getattr(image_properties.dominant_colors, 'colors', ())
+        if len(colors) == 0:
+            return ()
+        return [cls([ColorInformation.from_pb(color) for color in colors])]
```

Comment:
I can change it, but I would want to change the HTTP version to match at the same time. Should that be a different PR?